### ISSUE TITLE
DATASETS VARIABLES LIST

### ISSUE THREAD

**Issue by darkjhesus** [mz-automation/libiec61850#467]
Is it possible to discover from a IED all DataSets whit its variables list and Report Control with its datasets assigned like in commercial softwares?
 For example, the result of DataSets discover should be a list like is showed in the attached image and each datasets should has a list of logical nodes added to the dataset.

![Sin título](https://github.com/mz-automation/libiec61850/assets/18389488/910915f3-e3c8-4c8a-be70-28e22e2dd2a7)

**Comment by mzillgith** [comment]
Yes. You can do so with the discovery functions in the client API. E.g. to get the entries of a data set you can use the _IedConnection_getDataSetDirectory_ function.

**Comment by CarlosMC124** [comment]
Hi mzillgith, this function dont work in c#, maybe can you let me a one example please
